rc_building_model/htuse.py

```diff
@@ -122,3 +176,34 @@ def calculate_heat_loss_per_year(
     )
     heat_loss_kwh_for_heating_months.index = new_index
     return heat_loss_kwh_for_heating_months.sum(level=0).round()
+
+
+def calculate_useful_gains_per_year(
+    floor_area,
+    window_area,
+    mean_monthly_solar_gains=MEAN_MONTHLY_SOLAR_GAINS,
+    light_gain_value=8,  # Mean reference value taken from DEAP as 8W/m2 of gains
+):
+
+    solar_gain_kwh = (window_area / 6) * (
+        mean_monthly_solar_gains * UTILISATION_FACTOR
+    ).sum()  # Total Area divided by 6 to divide orientation
```

Comment:
Why 6?  Where does this come from?  Can you capture this as a variable instead?
```suggestion
    NAME_ME = 6
    solar_gain_kwh = (window_area / NAME_ME) * (
        mean_monthly_solar_gains * UTILISATION_FACTOR
    ).sum()  # Total Area divided by 6 to divide orientation
```